tests/unit/controllers/new-signup-test.js

```diff
@@ -0,0 +1,82 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | new-signup', function (hooks) {
+  setupTest(hooks);
+  test('testing current state after running their separate controller function', function (assert) {
+    let controller = this.owner.lookup('controller:new-signup');
+
+    assert.equal(
+      controller.currentStep,
+      'get-started',
+      'current step initially will be get-started'
+    );
+
+    controller.send('firstStep');
+    assert.equal(
+      controller.currentStep,
+      'firstName',
+      'current step updated to first name'
+    );
+
+    controller.send('secondStep');
+    assert.equal(
+      controller.currentStep,
+      'lastName',
+      'current step updated to lastName'
+    );
+
+    controller.send('thirdStep');
+    assert.equal(
+      controller.currentStep,
+      'username',
+      'current step updated to username'
+    );
+  });
```

Comment:
not written test for fourth and fifth step as windows.alert is not allowed in test, so the tests for them are going to fail